Source/Scene/Cesium3DTileset.js

```diff
@@ -2663,6 +2673,11 @@ function update(tileset, frameState, passStatistics, passOptions) {
     return false;
   }
 
+  var customShader = tileset.customShader;
```

Comment:
@lilleyse another thing is I don't like that the CustomShader now has GPU resources that A. need to be loaded in `update()` and destroyed in destroy(). and B. it means that either the Tileset or the Model is now responsible for cleanup depending on if this is 3D tiles or not... is there a better way we can do this with less mess?

Reply:
This is not necessary with the `TextureManager`, the implementation will only call update() once per frame. So the Model can always be responsible for updating the `CustomShader`